packages/vite/src/node/utils.ts

```diff
@@ -37,7 +37,10 @@ export function unwrapId(id: string): string {
 }
 
 export const flattenId = (id: string): string =>
-  id.replace(/(\s*>\s*)/g, '__').replace(/[\/\.:]/g, '_')
+  id
+    .replace(/(\s*>\s*)/g, '__')
+    .replace(/[\/:]/g, '_')
+    .replace(/[\.]/g, '___')
```

Comment:
```suggestion
    .replace(/[\/:]/g, '_')
    .replace(/[\.]/g, '__')
    .replace(/(\s*>\s*)/g, '___')
```
I think we can have `>` be `___` instead as it's less common. I've also sorted it to look a bit nicer 😬